chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -898,6 +898,7 @@ proxies.notification.associated.label	= Zotero automatically associated this sit
 proxies.notification.redirected.label	= Zotero automatically redirected your request to %1$S through the proxy at %2$S.
 proxies.notification.enable.button		= Enable…
 proxies.notification.settings.button	= Proxy Settings…
+proxies.notification.hide.button      = Don't Show Again
```

Comment:
Let's actually make this `general.dontShowAgain` above, since this will probably come in handy elsewhere. And this should be a proper typographical apostrophe.